src/Common/FunctionDocumentation.cpp

```diff
@@ -238,6 +240,7 @@ String FunctionDocumentation::categoryAsString() const
         {Category::Distance, "Distance"},
         {Category::EmbeddedDictionary, "Embedded Dictionary"},
         {Category::Geo, "Geo"},
+        {Category::GeoPolygon, "Geo Polygon"},
```

Comment:
Enum value "GeoPolygon" is conceptually a subset of enum value "Geo". Enum values are ideally on the same "level". I guess we should have only "Geo" for simplicity reasons.

We should also write them out: "Category::Geometry" instead of "Category::Geo".